statsmodels/genmod/families/family.py

```diff
@@ -431,6 +432,8 @@ class Gaussian(Family):
     links = [L.log, L.identity, L.inverse_power]
     variance = V.constant
 
+    # Gaussian family does not have safe_links since all links are safe
```

Comment:
Sounds good.  I used a tuple because isinstance requires a tuple not a list.  But I can convert the list to a tuple in the isinstance call.

Also added the checks to GEE.